@kiva/kv-components/vue/stories/KvLightbox.stories.js

```diff
@@ -56,6 +56,75 @@ const DefaultTemplate = (args, { argTypes }) => ({
 	},
 });
 
+const VeryShortTemplate = (args, { argTypes }) => ({
```

Comment:
New lightbox stress test stories, one with very little content, one with a lot of content